data/mods/Alternative_Zombie_Evolution/modinfo.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "alternative_zombie_evolution",
+    "name": "Alternative Zombie Evolution",
```

Comment:
I think the name is confusing. It doesn't alternates evolution, it just forbids evolved monsters to spawn in the beginning. Consider naming it something like "No advanced monsters in the beginning".